website/pages/_document.js

```diff
@@ -1,4 +1,4 @@
-import Document, { Head, Main, NextScript } from 'next/document'
+import Document, { Html, Head, Main, NextScript } from 'next/document'
```

Comment:
As soon as I upgraded `next`, a warning popped up about the need to use this `Html` component, so I've imported and used it per [`next`'s docs on custom `Documents`](https://nextjs.org/docs/advanced-features/custom-document).